packages/babel-compat-data/native-modules.js

```diff
@@ -1 +1,3 @@
-module.exports = require("./data/native-modules.json");
+module.exports = process.env.BABEL_8_BREAKING
+  ? require("./data/native-modules-next.json")
+  : require("./data/native-modules.json");
```

Comment:
The flag here will ship to end users because we didn't run Babel on these CJS proxies.

Reply:
Can we do this using rollup?